modules/vstudio/vslatest.lua

```diff
@@ -0,0 +1,58 @@
+--
+-- vslatest.lua
+-- Extend the existing exporters with support for Visual Studio 2019.
+-- Copyright (c) Jason Perkins and the Premake project
+--
+
+	local p = premake
+	local vstudio = p.vstudio
+	local latest = nil
+
+	function getLatestVisualStudioVersion()
+		-- Get the most recent version installed
+		if latest == nil then
+			local version = nil
+			if os.host() == 'windows' then
+				local ver, err = os.outputof('"C:\\Program Files (x86)\\Microsoft Visual Studio\\Installer\\vswhere.exe" -latest -property catalog_productLineVersion')
+				if err ~= nil and err == 0 then
+					version = 'vs' .. ver
+				end
+			end
+
+			if version == nil then
+				version = "vs2019"
+				print("No Visual Studio installation found using vswhere.exe. Defaulting to " .. _ACTION .. ".")
+			end
+
+			latest = p.action.get(version)
+		end
+	
+		return latest
+	end
+
+---
+-- Define the Visual Studio Latest export action.
+---
+
+	newaction {
+		-- Metadata for the command line and help system
+		
+		trigger	    = "vs-latest",
+		shortname   = "Visual Studio - Latest",
+		description = "Generate Visual Studio project files of latest installation",
```

Comment:
Nitpick, but maybe this would be clearer as something like "Generate project files for newest installed Visual Studio"? At the very least, should be "project files *for* latest" instead of "project files of latest". Otherwise looks okay to me.

Reply:
Fixed!